language/ir-testsuite/tests/move/transactions/multi_agent.mvir

```diff
@@ -0,0 +1,22 @@
+//! new-transaction
```

Comment:
Yes it can be written in the source language. I added this mvir test to make sure that our ir testsuite can deal with transactions with multiple signers(previously it couldn't). Since we are not supposed to add new IR tests, I will rewrite this in move lang and move the new test to move-lang tests.